src/components/content/myServices/MyServices.tsx

```diff
@@ -118,6 +127,20 @@ function MyServices(): React.JSX.Element {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [isDestroyingCompleted]);
 
+    useEffect(() => {
+        if (isPurgingCompleted) {
+            const timeoutId = setTimeout(() => {
+                setId('');
+                refreshData();
+            }, delayRefreshTableTime);
+
+            return () => {
```

Comment:
why do we need a timeout for this?  Is it not possible to simply execute 'refreshData()' whenever purging is successful.